devices.js

```diff
@@ -14374,10 +14401,19 @@ const devices = [
         model: 'MOSZB-140',
         vendor: 'Develco',
         description: 'Motion sensor',
-        supports: 'occupancy',
-        fromZigbee: [fz.ias_occupancy_alarm_1],
+        supports: 'occupancy, temperature, illuminance',
+        fromZigbee: [fz.temperature, fz.illuminance, fz.ias_occupancy_alarm_1],
         toZigbee: [],
-        exposes: [e.occupancy(), e.battery_low(), e.tamper()],
+        exposes: [e.occupancy(), e.battery_low(), e.tamper(), e.temperature(), e.illuminance_lux()],
```

Comment:
Are these changes expected?

Reply:
No, I had intended on submitting a separte PR for this, but did want to add illuminance and temperature reporting for this device (since it is supported).  I can update the title of this PR to cover both changes, or can resubmit as two separate PRs.   Which would you prefer?